src/lang.js

```diff
@@ -8,6 +8,40 @@
    * @words expect array
    *
    */
+
+  let setLangType = function (LangType) {
+    let currentLangType;
+    //  Check language type is present in congreaLanguages or not.
+    if (Object.prototype.hasOwnProperty.call(window.congreaLanguages,LangType)) {
+      currentLangType = LangType;
+    } else if (LangType.length >= 4) {
+      // check language by first two characters of langtype
+      LangType = LangType.substring(0, 2);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Assignment to function parameter 'LangType'.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5529949)